README.md

```diff
@@ -44,6 +44,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/j
 + [KiBot](https://github.com/INTI-CMNB/KiBot): KiCad automation utility.
 + [KiCad RF Tools](https://github.com/easyw/RF-tools-KiCAD): Footprints, wizards and round tracks, mask expander, via fencing
 + [Import-LIB-KiCad-Plugin](https://github.com/Steffen-W/Import-LIB-KiCad-Plugin): Import KiCad component libraries imported from Ultralibrarian and SnapEDA zipfiles.
++ [typeCAD](https://www.typecad.net): Programmatically create hardware designs with TypeScript. Create/reuse/import version-controlled packages between projects. 
```

Comment:
Can you please add https://github.com/typecad/typecad instead of the web page?

Reply:
Sure